documentation/assemblies/upgrading/assembly-upgrade-kafka.adoc

```diff
@@ -7,17 +7,19 @@
 
 To upgrade brokers and clients without downtime, you _must_ complete the Strimzi upgrade procedures in the following order:
 
-. . When upgrading from {ConvertAfterProductVersion} or earlier, update existing custom resources to support the `v1beta2` API version.
+. Update your Kubernetes cluster to a supported version.
```

Comment:
Emphasis on "supported" too! But articulating what is supported and not supported is difficult! For example, Strimzi 0.22 users wanting to upgrade to k8s v1.22 will need to migrate their Strimzi custom resources to `v1beta2` before the k8s upgrade since `apiextensions/v1beta1` custom resources are not supported in k8s v1.22. After doing the CRD migration, they can safely upgrade their k8s cluster.  So Strimzi 0.22 which is the last Strimzi version to use `apiextensions/v1beta1` is not fully supported on k8s 1.22.

Reply:
I've changed to _Make sure..._
And added a line for latest:

![image](https://user-images.githubusercontent.com/47596553/135084130-8d775353-b68a-4e44-9215-c8527c2ac99e.png)